docs/en/ingest-management/elastic-agent/elastic-agent-providers.asciidoc

```diff
@@ -148,38 +145,37 @@ Provides information about the current host. The available keys are:
 [[env-provider]]
 ==== Env Provider
 
-Provides access to the environment variables as key/values.
+Provides access to the environment variables as key and values.
 
-For example, if you set the variable foo:
+For example, set the variable `foo`:
 
 [source,shell]
 ----
 foo=bar elastic-agent run
 ----
 
-You can reference the environment variable as `${env.foo}`.
+The environment variable can be referenced as `${env.foo}`.
 
 [discrete]
 [[kubernetes_secrets-provider]]
 ==== Kubernetes Secrets Provider
 
 Provides access to the Kubernetes Secrets API.
 
-Provider needs a `kubeconfig` file so as to establish connection to Kubernetes API,
-or it can automatically reach the API if it runs in an inCluster environment (Agent runs as Pod).
+The provider needs a `kubeconfig` file to establish connection to the Kubernetes API.
+It can automatically reach the API if it's run in an inCluster environment (agent runs as Pod).
```

Comment:
Should pod be capitalized? And should agent be capitalized? The style guide didn't really give a clear direction for either of these, but I think this will need to be spot checked throughout based on your feedback.

Reply:
Agent should be capitalized. Pod should not.

Reply:
Agent is a product name, so we have an attribute that you can use. This way you don't even have to worry about it 😄. I've explained it in more detail in another comment.

```suggestion
It can automatically reach the API if it's run in an inCluster environment ({agent} runs as pod).
```